sunpy/net/dataretriever/sources/goes.py

```diff
@@ -54,6 +57,24 @@ def _get_goes_sat_num(self, date):
             raise ValueError('No operational GOES satellites on {}'.format(
                 date.strftime(TIME_FORMAT)))
 
+    def _get_time_for_url(self, urls):
+        times = []
+        for uri in urls:
+            uripath = urlsplit(uri).path
+
+            # Extract the yymmdd or yyyymmdd timestamp
+            datestamp = os.path.splitext(os.path.split(uripath)[1])[0][4:]
+
+            if int(datestamp) < 990115:
```

Comment:
is that 1999-01-15? or is the number of seconds since computer existed? a comment would help for people when look that in a confusing way.